### PR TITLE
Create a component to display upcoming events

### LINKED ISSUE #634: Create an example component to display upcoming event
**Is your feature request related to a problem? Please describe.**
A component is needed to display upcoming events.

**Describe the solution you'd like**
Design and create a new component in the examples/components.html page

**Describe alternatives you've considered**
N/A

**Additional context**
If you are a designer, feel free to create a design and send a screenshot.

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

The purpose of this PR is to fix #634 

## Goals

To create a component to display the upcoming event

## Approach

Created a component using cards and added custom CSS for that

### Screenshots

SEF_site_Components_-_2020-04-02_20 47 47

  
### Preview Link

https://pr-635-sef-site.surge.sh/examples/components.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
N/A

## Test environment
 
Ubuntu 19.04
Google Chrome

## Learning

N/A

### CODE REVIEW

File: src/main/webapp/assets/css/global-custom.css [hunk 1]
```diff
@@ -488,3 +488,16 @@ body{
 .faq .accordion .question .question-heading {
     padding-right: 30px;
 }
+
+/*upcoming events card*/
+
```
Comment (jayasanka-sack) [inline src/main/webapp/assets/css/global-custom.css]: Remove this space

File: src/main/webapp/examples/components.html [hunk 2]
```diff
@@ -2096,6 +2096,37 @@ <h5 class="title">
...
+                    <div class="card card-event shadow mb-1">
+                        <div class="card-body">
+                            <span class="h5 mr-1 card-event-date">15</span>
+                            <span class="small text-muted mr-4">MAY, SUN</span>
+                            <span class="mr-5"> 7.00 - 8.00 pm</span>
+                            <span class="font-weight-bold ml-1">Another onelive event</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/components.html:2128]: This should be in another row right?

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Solid design! 😍🔥

Just fix these two minor changes

File: src/main/webapp/examples/components.html (hunk 2 above)
Comment (Piumal1999) [reply src/main/webapp/examples/components.html:2128]: Oh yes😁

Comment (Piumal1999) [reply src/main/webapp/examples/components.html:2128]: fixed

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Piumal1999! 
I love this design! 🔥

**Review by anjula-sack — APPROVED** [review]
Awesome! @Piumal1999

**Pull request merged**